src/app/util/basic-types.h

```diff
@@ -57,5 +58,6 @@ typedef CHIPEndpointId EndpointId;
 typedef CHIPClusterId ClusterId;
 typedef CHIPAttributeId AttributeId;
 typedef CHIPGroupId GroupId;
+typedef CHIPCommandId CommandId;
```

Comment:
Whats the reason for also creating a `CHIPCommandId` type?

Reply:
We still have some headers that are being included from `.c` files that want to use these types.